onnx/defs/traditionalml/defs.cc

```diff
@@ -47,16 +47,16 @@ ONNX_ML_OPERATOR_SET_SCHEMA(
              "tensor(double)",
              "tensor(int64)",
              "tensor(int32)"},
-            "allowed types.")
+            ".")
         .Attr(
             "threshold",
-            "Values greater than this are set to 1, else set to 0",
+            "Values greater than this are replaced by 1, others by 0.<br>The default threshold is 0.0.",
```

Comment:
'Set’ is better than "Replace". The input tensor is NOT replaced. it's 0 or 1 in the output tensor.

Reply:
How about "are mapped to 1, others to 0" ?

Reply:
I went with 'mapped'